### DISCUSSION TITLE
Update cycle for marine and land zones: union of world country boundaries and EEZ's?

### DISCUSSION THREAD

**Discussion by brendan-ward** [lifewatch/marineregions-issues#15] [Q&A] [answered]
Is there an update planned for the EEZ / land union dataset to use EEZ version 12?

We use this alongside the EEZ boundaries to depict a country's full marine and terrestrial estate on the [Marine Protection Atlas](https://mpatlas.org/), so we've been unable to upgrade to EEZ version 12 without this.

Thank you very much for the EEZ and EEZ / land union datasets!  It has been very helpful to use these in our analyses.

**Comment by lawrencewhatley** [comment, marked as answer]
Hello, yes the updated dataset using EEZ v12 was released in October 2024! You can download it at https://www.marineregions.org/downloads.php.
